proto/v1/grafeas.proto

```diff
@@ -171,8 +171,10 @@ message Occurrence {
   // `projects/[PROJECT_ID]/occurrences/[OCCURRENCE_ID]`.
   string name = 1;
 
-  // Required. Immutable. The resource for which the occurrence applies.
-  Resource resource = 2;
```

Comment:
Why just use a simple string to replace resource?
I think **Resource** with a **content_hash** field is very useful in cases such as a SnapShot jar or some dependencies accidentally overwritten.

Reply:
cc @vtsao

Reply:
We hadn't fully fleshed out the Resource message and its use cases, so we decided to move back to a resource string. One day we might switch to a Resource message again.

This way the URL is more free form and providers can construct the URL in whichever way they want to represent the resource. For now if you're using content hashes, we recommend putting it in the URL, just like a docker image does.